tests/testthat/test-workflow-bbi.R

```diff
@@ -76,6 +81,7 @@ withr::with_options(list(rbabylon.bbi_exe_path = BBI_PATH,
   })
 
   test_that("copying model works and new models run correctly", {
+    # TODO: this test relies on model-examples-bbi/1.ctl being present
```

Comment:
this is created on line 44, outside of any other test in a "setup" section. Is this TODO intended to put this section inside an explicitly named `setup_` function? 

I guess my point is: many of our tests rely on creating some state at the beginning of the test file and relying on it for multiple different tests. This is different than the tests themselves being dependent on each other, which I _think_ is your concern here.

Reply:
Yes, exactly. This test depends on another test, which we should avoid. The point of adding the `TODO` was to remind us to think about how to isolate this test.